verifier-sdk/src/light_transaction.rs

```diff
@@ -1151,3 +1142,80 @@ impl<
         }
     }
 }
+
+#[cfg(all(target_os = "solana", feature = "custom-heap"))]
+pub mod custom_heap {
+    use anchor_lang::{
+        prelude::*,
+        solana_program::entrypoint::{HEAP_LENGTH, HEAP_START_ADDRESS},
+    };
+    #[global_allocator]
+    static A: BumpAllocator = BumpAllocator {
+        start: HEAP_START_ADDRESS as usize,
+        len: HEAP_LENGTH,
+    };
+
+    use std::{alloc::Layout, mem::size_of, ptr::null_mut, usize};
```

Comment:
Not sure why rustfmt didn't complain about this, but can we please keep the imports up?

Reply:
probably because it's target os solana